src/main/java/com/stuypulse/robot/commands/ShooterControlCommand.java

```diff
@@ -0,0 +1,71 @@
+package com.stuypulse.robot.commands;
+
+import com.stuypulse.robot.Constants;
+import com.stuypulse.robot.subsystems.Shooter;
+import com.stuypulse.stuylib.input.WPIGamepad;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ShooterControlCommand extends CommandBase {
+    public Shooter shooter;
+    public WPIGamepad gamepad;
+    public double targetVelocity;
+
+    public ShooterControlCommand(Shooter shooter, WPIGamepad gamepad) {
+        this.shooter = shooter;
+        this.gamepad = gamepad;
+        addRequirements(shooter);
+    }
+
+    @Override
+    public void execute() {
+        // If up dpad button is pressed, shoot from far (Steph Curry)
+        if (gamepad.getRawDPadUp()) {
+            targetVelocity = Constants.SHOOT_FROM_FAR_RPM;
+
+            shooter.retractHoodSolenoid();  
+            shooter.runShooter();
+            shooter.runFeeder();
+            
+            if (shooter.getCurrentShooterVelocityInRPM() > targetVelocity) {
```

Comment:
This is going to keep in rumbling once the RPM is reached, you should have a check to make that not happen

Reply:
I would have a Boolean that resets every time the driver changes the shooting mode